app/assets/javascripts/rentals/index.js

```diff
@@ -6,6 +6,7 @@ $(document).ready(function() {
 		eventClick: function(calEvent, jsEvent, view) {
 			var win = window.open(calEvent.url, '_blank');
 			win.focus();
+			return false;
```

Comment:
From the Documentation I linked:
"Normally, if the Event Object has its url property set, a click on the event will cause the browser to visit the event's url (in the same window/tab). Returning false from within your function will prevent this from happening."

It looks like by returning false it breaks out of the event, stopping the code from executing that opens the url in the current window.